drivers/scheduler/k8s/k8s.go

```diff
@@ -340,6 +340,67 @@ func (k *K8s) AddNewNode(newNode corev1.Node) error {
 	return nil
 }
 
+func (k *K8s) SetGkeConfig(kubeconfigPath string, gkeCredString string) error {
```

Comment:
Is it possible to call `func (k *K8s) SetConfig(kubeconfigPath string) error` inside this?

Reply:
yes